config/prow/autobump-config/prow-component-autobump-config.yaml

```diff
@@ -14,6 +14,8 @@ includedConfigPaths:
 excludedConfigPaths:
   - "config/prow-staging"
 targetVersion: "latest"
+labels:
+  - "auto/trusted-pullrequest" # This label is used by tide for identifying trusted PR
```

Comment:
Please also create this label via the label_sync [config](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml).

Reply:
Done